python/pyfunc-server/run.sh

```diff
@@ -15,4 +15,4 @@
 
 
 source activate merlin-model
-python -m pyfuncserver --model_dir ./model --model_name $MODEL_NAME --workers $WORKERS
+python -m pyfuncserver --model_dir ./model
```

Comment:
why removing number of WORKERS

Reply:
`WORKERS` is now accepted via environment variable (see [config.py](https://github.com/gojek/merlin/blob/b13d18ec09faba79ce8819902464c3d4da3f2ef3/python/pyfunc-server/pyfuncserver/config.py))